README.md

```diff
@@ -5,29 +5,31 @@
 
 The Rust-ML working group is currently at an early stage of development, and is still deciding on several choices at an organizational level. This document will be updated accordingly as those decisions are made.
 
-### How can you contribute?
+## What do we do?
 
-Come say hello, read about, and weigh in on our [discussions](https://github.com/rust-ml/discussion)! We currently have several ongoing threads regarding upcoming scope of work, organizational structures, and more.
+In general, the Rust-ML working group's goals are to:
 
-The Rust-ML group has a Zulip channel. Feel free to take a look and say hi!
+- Create spaces where people interested in doing machine learning in Rust can meet, ask questions, and discuss their work
+- Help organize support for Rust libraries, tools, and projects in the machine learning domain
+- In rare instances, to assist with or assume the maintainership of projects important to the wider community
+
+## How can you contribute?
+
+The Rust-ML group has a Zulip chat. Feel free to take a look and say hi!
 - https://rust-ml.zulipchat.com
 
-### Organization
+We also host a video chats every two weeks. Meeting minutes can be found in the [Meetings](https://github.com/rust-ml/Meetings) repository.
```

Comment:
I would also mention meetings are normally announced in the Zulip

Reply:
Good suggestion, will do